src/panels/config/backup/ha-config-backup-settings.ts

```diff
@@ -203,6 +206,29 @@ class HaConfigBackupSettings extends LitElement {
                   `
                 : nothing}
             </div>
+            <div class="card-actions">
+              <a
+                href=${documentationUrl(this.hass, "/integrations/#backup")}
+                target="_blank"
+                rel="noreferrer"
+              >
+                <ha-button>
+                  <ha-svg-icon slot="icon" .path=${mdiOpenInNew}></ha-svg-icon>
+                  ${this.hass.localize(
+                    "ui.panel.config.backup.settings.locations.more_locations"
+                  )}
+                </ha-button>
+              </a>
```

Comment:
You need to set `text-decoration: none` to don't have this strange line here

![image](https://github.com/user-attachments/assets/691e8757-15f7-4eaf-9b3e-3b16a74fe2b2)